README.md

```diff
@@ -297,79 +297,81 @@ And for [Mendix \< 6.6](https://docs.mendix.com/releasenotes/desktop-modeler/6.6
 * `BLOBSTORE/mx-buildpack/jre-8u51-linux-x64.tar.gz`
 * `BLOBSTORE/mx-buildpack/jdk-8u51-linux-x64.tar.gz`
 
-### Certificate Management
+### Managing Certificate Authorities
 
 To import Certificate Authorities (CAs) into the Java truststore, use the `CERTIFICATE_AUTHORITIES` environment variable.
 
-The contents of this variable should be a concatenated string containing a the additional CAs in PEM format that are trusted. Example:
+The contents of this variable should be a concatenated string containing a the additional CAs in [PEM format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail) that are trusted.
```

Comment:
Should we mention that the certificate chain needs to be added if the certificate to check is signed by an intermediate?